operations/package_repository_management/Package_Repository_Management_with_Nexus.md

```diff
@@ -42,6 +43,123 @@ Specified repositories have been refreshed.
 The `-G` option is used in this example to disable GPG checks. However, if the named repository is properly signed, it is not recommended to use the
 `-G` option.
 
+### Creating a new RPM repository
```

Comment:
Break this section up into three or more.

1. Authenticating to Nexus: this will have the "Use the following function to get the nexus local admin account" step, this way other procedures or future steps can quickly refer to this common step
2. Creating a repo: this is where the "Create and export the repository name" and the rest of the new steps will exist
3. Adding/Removing a repo from Zypper

It might be good to add a step for removing a repo from Nexus.
```
function nexus-delete-repo() {
    local name="${1:-}"
    local error
    echo >&2 "Deleting $name ..."
    if ! curl \
    -f \
    -L \
    -v \
    -u "${NEXUS_USERNAME}":"${NEXUS_PASSWORD}" \
    -X DELETE \
    "${NEXUS_URL}/service/rest/v1/repositories/${name}" ; then
        error=1
    fi
    if [ "$error" -ne 0 ]; then
        echo >&2 'Errors found.'
    else
        echo 'Done'
    fi
    return "$error"
}
```

Lastly, I think the steps should be an ordered list with the code blocks indented over. This changes it into a procedural flow, each step

Reply:
Added the changes as requested. Please review and add any new notes to the changes.